modules/markup/markup.go

```diff
@@ -62,10 +62,12 @@ func Type(filename string) string {
 // IsReadmeFile reports whether name looks like a README file
 // based on its name.
 func IsReadmeFile(name string) bool {
-	name = strings.ToLower(name)
 	if len(name) < 6 {
 		return false
-	} else if len(name) == 6 {
+	}
+
+	name = strings.ToLower(name)
+	if len(name) == 6 {
 		return name == "readme"
 	}
 	return name[:7] == "readme."
```

Comment:
Couldn't this be simplify by just `return name == "readme" || name[:7] == "readme."` and don't remove if len(name) == 6 bracket ?